.flake8

```diff
@@ -0,0 +1,3 @@
+[flake8]
```

Comment:
We are currently setting configuration for flake8 in the `setup.cfg`. You're more than welcome to copy/paste that config into the `.flake8` file if you prefer it there (I think I do, actually).

Reply:
Done, thanks for pointing that out.